sympy/stats/crv_types.py

```diff
@@ -1172,7 +1192,15 @@ def FisherZ(name, d1, d2):
 class FrechetDistribution(SingleContinuousDistribution):
     _argnames = ('a', 's', 'm')
 
-    set = Interval(0, oo)
+    @staticmethod
+    def check(a, s, m):
+        _value_check(a > 0, "a must be positive")
```

Comment:
Use actual parameter names. For eg. Here it is `shape` parameter. Same goes for `s` and `m` in the next two lines.